renderer/viewer/three/world/vrHud.ts

```
@@ -0,0 +1,129 @@
+  attachToVRCamera(vrCameraGroup: THREE.Object3D) {
+    // Add HUD to the VR camera group so it follows the player's view
+    vrCameraGroup.add(this.hudGroup)
+  }
+  
+  detachFromVRCamera(vrCameraGroup: THREE.Object3D) {
+    vrCameraGroup.remove(this.hudGroup)
+  }
+  
+  update() {
+    // Get player data
+    const bot = (window as any).bot
```

ESLint

[error] Use object destructuring.

(prefer-destructuring)